ras/sosreport.py

```diff
@@ -207,12 +207,13 @@ def test_others(self):
         if os.listdir(dir_name) == []:
             self.is_fail += 1
             self.log.info("--no-report option failed")
-        file_list = self.params.get('file_list', default=['proc/device-tree/'])
-        for files in file_list:
-            file_path = os.path.join(dir_name, files)
-            if not os.path.exists(file_path):
-                self.is_fail += 1
-                self.log.info("%s file/directory not created" % file_path)
+        if 'powerpc' in cpu.get_arch():
+            file_list = self.params.get('file_list', default=['proc/device-tree/'])
+            for files in file_list:
+                file_path = os.path.join(dir_name, files)
+                if not os.path.exists(file_path):
+                   self.is_fail += 1
+                   self.log.info("%s file/directory not created" % file_path)
```

Comment:
@kalpanashtty please take care of indentation issue in 215 and 216 line number , better run autope8  and see it will resolve issue

Reply:
@PraveenPenguin corrected the lines 215 & 216, can you please check?